tests/regression/test_voting_acc.py

```diff
@@ -1,23 +1,58 @@
 """
-Tests for lido staking limits
+Tests for lido aragon voting app
 """
-from typing import Tuple, Optional
-
-import pytest
 
-from brownie import MockCallTarget, accounts, chain, reverts
+from typing import Tuple, Optional
+from enum import Enum
+from brownie import MockCallTarget, accounts, chain, reverts, interface, ZERO_ADDRESS
 from brownie.network.transaction import TransactionReceipt
 from utils.voting import create_vote, bake_vote_items
 from utils.config import LDO_VOTE_EXECUTORS_FOR_TESTS
 from utils.evm_script import EMPTY_CALLSCRIPT
 from utils.config import contracts
+from utils.test.extra_data import VoterState
+
+import pytest
+
+
+def filter_logs(logs, topic):
+    return list(filter(lambda l: l["topics"][0] == topic, logs))
 
 
 @pytest.fixture(scope="module")
 def call_target():
     return MockCallTarget.deploy({"from": accounts[0]})
 
 
+@pytest.fixture(scope="module")
+def test_trp_escrow(trp_recipient, ldo_holder):
+    contracts.ldo_token.approve(
+        contracts.trp_escrow_factory.address, 1_000_000_000_000_000_000, {"from": accounts.at(ldo_holder, force=True)}
```

Comment:
The `10 ** 18` or `Wei('1 ether')` notation is better for reading — no need to count zeros in the number.

Reply:
Fixed in 98c41312c4dcbf45286acc799ee16c3e46be578a.